x-pack/platform/plugins/shared/fleet/server/services/epm/packages/get_bulk_assets.ts

```diff
@@ -86,12 +86,17 @@ export async function getBulkAssets(
         }
       }
 
+      let title: string | undefined;
+      if (types[obj.type]?.management?.getTitle) {
```

Comment:
if `types[obj.type]?.management?.getTitle` is undefined, shouldn't we fall back to `obj.attributes?.title`?